src/app/components/Book/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+import { COLORS } from '@constants/colors';
+import { SIZES } from '@constants/fonts';
+
+export const styles = StyleSheet.create({
+  container: {
+    backgroundColor: COLORS.WHITE,
+    width: 375,
+    height: 102,
+    borderRadius: 5,
+    marginHorizontal: 20,
+    marginVertical: 10,
+    paddingLeft: 30,
+    justifyContent: 'center'
+  },
+  content: {
+    flexDirection: 'row'
+  },
+  title: {
+    fontSize: SIZES.MEDIUM,
+    color: COLORS.GREY,
+    fontWeight: 'bold'
+  },
+  author: {
+    fontSize: SIZES.SMALL,
+    color: COLORS.GREY,
+    fontWeight: '300'
```

Comment:
This is weird => `fontWeight: 300`

Reply:
I think that's okay, let's talk @sfernandez11

Reply:
I double check and it seems that this style attribute expects a string.